django-backend/fecfiler/forms/migrations/0001_initial.py

```diff
@@ -0,0 +1,185 @@
+# Generated by Django 3.2.11 on 2022-02-09 22:00
+
```

Comment:
If we really want to commit migrations files, I think these are out of scope of the ticket regardless.